### ISSUE TITLE
can not set n_chains 

### ISSUE THREAD

**Issue by LoreMJ** [statdivlab/paramedic#12]
I can only run run_paramedic when n_chains = 1. Other number of chains will report error!

**Comment by bdwilliamson** [comment]
Thanks for using `paramedic`, and for letting us know about this error! 

I have just implemented a fix for this; however, I am working on several other feature additions at the moment, so I will not be releasing a package update for a bit. 

In the meantime, you can work around this error to use multiple chains by passing in your own list of initial values. For example, if you want to use 2 chains, (with random initial values), you can use the following code:
```{r}
# creates a list of length 2
inits <- replicate(2, list(init = "random"), simplify = FALSE)
```
and pass this in using the argument `inits_lst` in `run_paramedic`.

**Comment by bdwilliamson** [comment]
Fixed in b111946f65 (v0.1.2)
